src/content/docs/en/guides/integrations-guide/preact.mdx

```diff
@@ -150,6 +151,22 @@ Check out the [`pnpm` overrides](https://pnpm.io/package_json#pnpmoverrides) and
 Currently, the `compat` option only works for React libraries that export code as ESM. If an error happens during build-time, try adding the library to `vite.ssr.noExternal: ['the-react-library']` in your `astro.config.mjs` file.
 :::
 
+### devtools
+
+<p><Since pkg="preact" v="3.3.0" /></p>
+
+You can enable [Preact devtools](https://preactjs.github.io/preact-devtools/) in development by setting `devtools: true` in your `preact()` integration config:
```

Comment:
```suggestion
You can enable [Preact devtools](https://preactjs.github.io/preact-devtools/) in development by passing an object with `devtools: true` to your `preact()` integration config:
```
Just looked at the section above, and this is closer to the wording used there, so I'll change that here and in the other PR!